lucene/core/src/java/org/apache/lucene/index/SortedDocValuesWriter.java

```diff
@@ -79,11 +78,6 @@ public void addValue(int docID, BytesRef value) {
     lastDocID = docID;
   }
 
-  @Override
-  public void finish(int maxDoc) {
-    updateBytesUsed();
-  }
```

Comment:
I wonder if there are undesired consequences of dropping this?

Reply:
It was always called either immediately before `flush` or `getDocComparator`, so it seemed to make sense to just fold it directly into those methods.